### PR TITLE
Align dir/file structures with LSP-css

### PR DESCRIPTION
**Pull request by jfcherng**
Do the same thing https://github.com/sublimelsp/LSP-css/pull/13

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 Предраг Николић
+Copyright (c) 2019-2020 Предраг Николић
```
Comment (predragnikolic) [inline LICENSE:3]: Feel free to put SublimeLSP instead of my name. GitHub probably put my name when I generated the License.

Comment (jfcherng) [reply LICENSE]: I guess https://github.com/sublimelsp/LSP-html/pull/10/commits/47cd077aedc6df8dcebd82023012f1c5d7e29af1 would make both side happy :smiley: ?

**Comment by predragnikolic** [comment]
I ran into the same problem as with the LSP-css PR. (I thought that it was my mistake, here is what I did).
I check out your branch.
I saw in the status bar the the server is installing.
I saw a message to restart sublime.
I restarted sublime. 
I opened an html file and saw this:

image

```
lsp-html: internal/modules/cjs/loader.js:960
lsp-html:   throw err;
lsp-html:   ^
```

Manually deleting the cache directory fixes the problem...

I wonder if people can run into this issue with PC. 

I installed LSP-css v1.0.0, 
and I did a release for the LSP-css, v1.0.1
I am waiting for PC to discover the v1.0.1 update and check if the problem exist.

**Comment by jfcherng** [comment]
IIRC, when upgrade, the plugin gets unloaded first, which will delete the cache automatically.

**Comment by predragnikolic** [comment]
I have a theory.
When I checkout your branch, 
The plugin_loaded event got fired, 
which caused LspHtmlPlugin.setup() to run, 
but LspHtmlPlugin.cleanup() wasn't run before that.
ending up with mixed up dependencies in cache/LSP-html/node_modules?

**Comment by predragnikolic** [comment]
Everything works fine when upgrading via Package Control, so I guess you are right:

>  when upgrade via PC, the plugin gets unloaded first.

**Comment by rchl** [comment]
This update uses different directory for the server (`LSP-html/language-server` while previously it was `LSP-html/vscode-html`). So the issue is not due to mixed dependencies.

I've noticed though that, for things to work properly, you need one more restart of ST than you would expect. I'll check that out.

**Comment by rchl** [comment]
OK, so just after switching to the new branch, `sublime.find_resources` sees those (filtered a bit by me):
```
Packages/LSP-html/.DS_Store
Packages/LSP-html/dependencies.json
Packages/LSP-html/LICENSE
Packages/LSP-html/LSP-html.sublime-commands
Packages/LSP-html/LSP-html.sublime-settings
Packages/LSP-html/Main.sublime-menu
Packages/LSP-html/plugin.py
Packages/LSP-html/README.md
Packages/LSP-html/tox.ini
Packages/LSP-html/vscode-html/.DS_Store
```
So that's not the contents of the new branch. And neither the old one as some files are already missing.

I don't think it's the bug, it's just that we can't rely on that API being in sync with files on disk always as it relies on a database that is managed asynchronously.

I doubt it would be a problem for normally installed packages as then they are in a zip file and once the new `py` file is running, the database should be up to date already.

**Comment by jfcherng** [comment]
Weird. I am getting the same loader error today.

```
lsp-html: internal/modules/cjs/loader.js:969
lsp-html:   throw err;
lsp-html:   ^
```

And deleting the `LSP-html` from the cache dir doesn't help.

**Comment by rchl** [comment]
you should look at the what is there in the LSP-html directory. Probably the actual source is missing.

**Comment by jfcherng** [comment]
Turns out that `typescript` is not installed as a non-dev dependency.

https://github.com/sublimelsp/LSP-html/blob/fa91b349f6362b16ffb7b2ab85f810ec5f3230fe/language-server/package.json#L25

**Pull request merged**